tardis/plasma/standard_plasmas.py

```diff
@@ -76,9 +76,12 @@ def assemble_plasma(config, model, atom_data=None):
 
         try:
             atom_data = atomic.AtomData.from_hdf(atom_data_fname)
-        except TypeError:
-            raise ValueError('Error might be from the use of an old-format of the atomic database,\n please see '+ \
-            'https://github.com/tardis-sn/tardis-refdata/tree/master/atom_data for the most recent version.')
+        except TypeError as e:
+            print (e, 'Error might be from the use of an old-format of the atomic database, \n' +\
```

Comment:
you don't need the plus. every string in parenthesis is automatically concatenated.

Reply:
Oh I see, noted, thanks!